views/settings/page-sections/dashboard.php

```diff
@@ -81,10 +81,10 @@
 						<span class="wpr-infoAccount wpr-isValid" id="wpr-account-data">
 							<?php echo esc_html( $data['customer_data']['license_type'] ); ?>
 						</span>
-						<?php if ( ! $data['customer_data']['is_from_one_dot_com'] ) : ?>
+						<?php if ( $data['customer_data']['is_from_one_dot_com'] ) : ?>
 							<span>
-								<?php echo esc_html( 'with ' ); ?> 
-								<img src="<?php echo esc_url( WP_ROCKET_ASSETS_IMG_URL . 'one-com-logo.svg' ); ?>" width="80" alt="Logo One.com">
+								<?php echo esc_html_e( 'with', 'rocket' ); ?> 
```

Comment:
```suggestion
								<?php esc_html_e( 'with', 'rocket' ); ?> 
```

`esc_html_e()` is already echoing the data, so you don't need it anymore